logstash-core/lib/logstash/api/lib/app/modules/stats.rb

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+require "app"
+
+module LogStash::Api
+  class Stats < BaseApp
+
+    helpers AppHelpers
+
+    # return hot threads information
+    get "/jvm" do
+      command = factory.build(:memory_command)
+      jvm_payload = {
+        :timestamp => command.started_at,
+        :uptime_in_millis => command.uptime,
```

Comment:
I'd like to question the decision to put the uptime in /jvm instead of /. this value will not change and is not related to jvm